test/test_torch.py

```diff
@@ -15293,6 +15249,24 @@ def test_complex_unsupported(self, device, dtype):
         with self.assertRaises(RuntimeError):
             torch.trunc(inp)
 
+    @dtypes(torch.long)
+    def test_abs_big_number(self, device, dtype):
+        bignumber = 2 ** 31 + 1
+        res = torch.tensor([bignumber], device=device, dtype=dtype)
+        self.assertGreater(res.abs()[0], 0)
+
+    @dtypes(torch.float, torch.double)
+    def test_abs_signed_zero(self, device, dtype):
+        # Both abs(0.0) and abs(-0.0) should result in 0.0
+        size = 128 + 1  # pick a large enough number with remainder so that
+        # both vectorized and nonvectorized op is tested
+        inp = torch.zeros(size, device=device, dtype=dtype)
```

Comment:
Nit nit: torch.fill fill_value=-0.?

Reply:
I'm filling with + and - zeros to make sure they both behave as expected.